README.md

```diff
@@ -1,3 +1,7 @@
+## Do not merge
+
+Hey, I can help you write this plugin if you are open to it, or at least help diagnose issues. You'll need to enable GitHub Issues for me to easily be able to communicate with you, though. Thank you for doing the work to add 2FA to this plugin, because obviously the original is now broken. I was going to add 2FA SMS support via Twilio as an option when I came across your fork. Would you like to work together? All good if not, have a great day.
+
```

Comment:
the message

Reply:
Hello, yes if you'd like to contribute feel free. [arlo-api](https://github.com/wo-d/arlo-api) is the underlying library which is doing the authentication and communication with Arlo services.

I would like to point out that this library and the arlo-api are very bare bones, and I've removed almost all of the existing functionality.

Reply:
@wo-d sounds good -- i would love to add Security Keypad support and then help with cameras, eventually, but the security keypad is my main need since the cameras already work with HomeKit natively.